mixer/adapter/svcctrl/utils.go

```diff
@@ -0,0 +1,154 @@
+// Copyright 2017 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package svcctrl
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"time"
+
+	pbtypes "github.com/gogo/protobuf/types"
+	rpc "github.com/googleapis/googleapis/google/rpc"
+)
+
+const (
+	apiKeyPrefix = "api_key:"
+
+	logErr    = 4
```

Comment:
do these align with other logging levels? Generally, `2` is being used for info logging, with `4` for debug.